tests/CourseDashboardTests.cpp

```diff
@@ -1,8 +1,29 @@
 #include <gtest/gtest.h>
 #include "CourseDashboard.hpp"
+#include "User.hpp"
+
+struct CourseDashboardTests : public ::testing::Test
+{
+    CourseDashboard CourseDashboard_;
+    User testUser_;
+
+    CourseDashboardTests()
+    {
+        CourseDashboard_.createUser(User("Stefan",
+                                       "SBatory",
+                                       "wieczorowa",
+                                       "sbatory",
+                                       "sbatory"));
+        testUser_ = User("Jan",
+                         "JanKowalski",
+                         "weekendowa",
+                         "jkowalski",
+                         "jkowalski");
+    }
+};
 
 
-TEST(CourseDashboardTests, canConstructCourseDashboard)
+TEST_F(CourseDashboardTests, canConstructCourseDashboard)
 {
     CourseDashboard cd;
 }
```

Comment:
Looks like not tested :( You must provide unit tests.

Reply:
In progress

Reply:
Current approach partial implementation in https://github.com/coders-school/CourseDashboard/pull/14/commits/53fed699000193c9997dba86926c56c3df846360